docs/sites.yml

```
@@ -10075,6 +10075,16 @@
     - Library
   built_by: Gilbish Kosma
   built_by_url: https://www.gil20.me/
+  - title: Monolit
```

```suggestion
- title: Monolit
```

This should probably fix it

---

Hi, I've just committed a fix as you suggested.